cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java

```diff
@@ -235,11 +247,51 @@ public void aggregate(NGSIEvent cygnusEvent) {
                 
                 LOGGER.debug("[" + getName() + "] Processing context attribute (name=" + attrName + ", type="
                         + attrType + ")");
-                Document doc = createDoc(recvTimeTs, entityId, entityType, attrName, attrType, attrValue);
+                
+                Document doc;
+				if(attrMetadataStore.equals("true")){
```

Comment:
I used the check style but no error has been showed, may be I miss configured?
![checkstyleconfig](https://cloud.githubusercontent.com/assets/11886594/23742243/662d6a72-04ac-11e7-940a-92307e8e6b31.png)